frame/asset-conversion/src/benchmarking.rs

```diff
@@ -17,8 +17,6 @@
 
 //! Asset Conversion pallet benchmarking.
 
-#![cfg(feature = "runtime-benchmarks")]
-
```

Comment:
Why's this removed? I notice it exists in most other `benchmarking.rs` files.

Reply:
(see below - sorry was on email when replying)